editor/inserter/menu.js

```diff
@@ -58,37 +56,66 @@ class InserterMenu extends Component {
 		} );
 	}
 
-	selectBlock( name ) {
+	selectBlock( blockType ) {
 		return () => {
-			this.props.onSelect( name );
+			this.props.onSelect( blockType.name );
+			recordUsage( blockType );
```

Comment:
What about using the `INSERT_BLOCK` action instead, which should already be firing?